cfy_manager/components/restservice/restservice.py

```diff
@@ -158,8 +162,7 @@ def _verify_restservice(self):
         there's a good chance everything is configured correctly.
         """
         rest_port = config[RESTSERVICE]['port']
-        url = 'http://{0}:{1}/api/v2.1/blueprints'.format('127.0.0.1',
-                                                          rest_port)
+        url = 'http://{0}:{1}/api/v1/version'.format('127.0.0.1', rest_port)
```

Comment:
Why v1?

Reply:
I forgot to change it back to v2.1/blueprints as we are not verifying the REST service at all during a join operation.
Reverted.